cg/nuget/cgmanifest.json

```diff
@@ -50,7 +50,7 @@
       "Component": {
         "Type": "NuGet",
         "NuGet": {
-          "Name": "Bond.CSharp",
+          "Name": "Bond.Core.CSharp",

```

Comment:
to fix "outdated cgmanifest file" validation failures, please pull upstream master into your branch and regenerate this file by running `bxl /cacheGraph- /phase:Schedule`

Reply:
since you didn't change any NuGets in this PR, the same you can achieve if you run
```
git checkout upstream/master -- cg/nuget/cgmanifest.json
git ci -am "revert cg/nuget/cgmanifest.json file"
git push origin
```